lerobot/common/policies/act/modeling_act.py

```diff
@@ -38,7 +38,13 @@
 from lerobot.common.policies.normalize import Normalize, Unnormalize
 
 
-class ACTPolicy(nn.Module, PyTorchModelHubMixin):
+class ACTPolicy(
+    nn.Module,
+    PyTorchModelHubMixin,
+    library_name="lerobot",
+    repo_url="https://github.com/huggingface/lerobot",
+    tags=["robotics", "act-policy"],
```

Comment:
Personal preference but I'd rather set a single tag `act-policy` (and same for the other policies). Feels more self-explanatory to me + in the UI it will allow to click on the `act-policy` tag to list all models with it. If split in two tags, you can filter models with `act` but some might not even be policies or robotics related. Especially in the case of `diffusion-policy` where "diffusion" alone can mean everything.

Reply:
Ha I see, I updated my suggestion. Thanks for the tip.